packages/material-ui/src/MenuItem/MenuItem.test.js

```diff
@@ -8,6 +8,7 @@ import ListItemSecondaryAction from '../ListItemSecondaryAction';
 import MenuItem from './MenuItem';
 
 describe('<MenuItem />', () => {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
why note remove the unused variables instead of adding the eslint rule? This would remove let shallow and shallow = createShallow({ dive: true }); from the before block

Reply:
Not sure when this was added. we should remove.

Reply:
```suggestion
```